cmd/tke-installer/app/installer/manifests/thanos/thanos.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v1
 data:
-  thanos.yaml: {{ .ThanosYaml }}
+  thanos.yaml: { { .ThanosYaml } }
```

Comment:
Official examples of template use "{{", not "{ {". I'm not sure whether "{{" will be rendered as expected.  I recommend to use what official examples use.